ui/src/app/components/apps-panel/utils.tsx

```diff
@@ -119,6 +119,15 @@ export const defaultSASCreateRequest: CreateAppRequest = {
   autodeleteThreshold: DEFAULT_AUTODELETE_THRESHOLD_MINUTES,
 };
 
+// This is not the best place for this, but it's causing test to pass, we need to rethink utils/user-app-utils
+export const appMinDiskSize: Record<AppType, number> = {
+  ['CROMWELL']: 50,
```

Comment:
Use `AppType.CROMWELL` instead.  Could be related to the problems you're having?

Where would you prefer to put this?

Reply:
Ahh sorry this was missed i was debugging compilation error

Reply:
what does the comment mean here?  where is a better place for this?

Reply:
user-app-utils, where most of constants are defined.